src/ansys/fluent/core/solver/flobject.py

```diff
@@ -62,6 +62,8 @@
     PyFluentDeprecationWarning = FutureWarning
     PyFluentUserWarning = UserWarning
 
+import ansys.fluent.core as pyfluent
```

Comment:
It's a bit unfortunate that we now need this `import` for `search()`. However, it isn't a true coupling to the `pyfluent` package. Eventually, we would move `flobject.py` outside of `solver` and it can do a relative import of `search`, which is just another generic thing.